kubernetes/chart/zulip/values-local.yaml.example

```diff
@@ -0,0 +1,58 @@
+# Replace each occurrence of "set-secure-password" with a different random
+# password.
+# Replace each occurrence of "zulip.example.com" with the domain you want Zulip
+# to be reached on.
+
+# Supports OIDC and the below `create_realm` script
+image:
+  repository: open.greenhost.net:4567/stackspin/stackspin/zulip
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "f60b8cc"
+ingress:
+  enabled: true
+  annotations:
+    # Get certificates with cert-manager
+    kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: zulip.example.com
+      paths:
+      - path: /
+  tls:
+    - secretName: zulip-tls-secret
+      hosts:
+        - zulip.example.com
+
+memcached:
+  memcachedPassword: set-secure-password
```

Comment:
Even though I see your good intention in the comment at the top of this file I think it's not a good practice to set default passwords, even in a `values-local.yaml.example` file. Please comment them at least so users won't make the mistake of copying this file and forgetting to change a default password.